compiler/luci/pass/src/RemoveDuplicateConstPass.cpp

```diff
@@ -21,24 +21,29 @@
 namespace
 {
 
-template <loco::DataType DT> bool is_equal_consts(luci::CircleConst *left, luci::CircleConst *right)
+bool compare_quant_params(luci::CircleConst *left, luci::CircleConst *right)
 {
-  const auto left_rank = left->rank();
-  const auto right_rank = right->rank();
-
   const auto left_quant_param = left->quantparam();
   const auto right_quant_param = right->quantparam();
 
   if (left_quant_param != nullptr and right_quant_param != nullptr)
   {
     if (left_quant_param->scale != right_quant_param->scale or
-        left_quant_param->quantized_dimension != right_quant_param->quantized_dimension)
-      return false;
-
-    if ((left->dtype() == loco::DataType::U8 or left->dtype() == loco::DataType::S8) and
-        left_quant_param->zerop != right_quant_param->zerop)
+        left_quant_param->quantized_dimension != right_quant_param->quantized_dimension or
+        left_quant_param->zerop != right_quant_param->zerop or
+        left_quant_param->min != right_quant_param->min or
+        left_quant_param->max != right_quant_param->max)
```

Comment:
It would be better to check quantparam is the same, and then return true.

This function will return true even if left has quantparam while right does not have quantparam.

Reply:
> This function will return true even if left has quantparam while right does not have quantparam.
 Fixed it, thanks